x-pack/metricbeat/module/gcp/pubsub/_meta/data_topic.json

```diff
@@ -12,25 +12,112 @@
         "module": "gcp"
     },
     "gcp": {
-        "labels": {
-            "resource": {
-                "topic_id": "test-ks"
-            }
-        },
         "pubsub": {
             "topic": {
                 "message_sizes": {
-                    "bucket_options": {
-                        "Options": {
-                            "ExponentialBuckets": {
-                                "num_finite_buckets": 16,
```

Comment:
We can keep this as well ?

Reply:
The old version was wrong, it's not how an event would look like. The updated sample document is fine without the `bucket_options`.